src/chapters/objects-and-math/modifyingobjects.rst

```diff
@@ -0,0 +1,145 @@
+Working with Objects
+====================
+
+Accessing Properties
+--------------------
+
+When using objects, programmers oftentimes want to retrieve or change the value of one of the properties.
+To access the value of a property, you will need the object's name and the key of the property.
+
+Programmers have two ways to access the value of property:
+
+1. Bracket syntax
+2. Dot notation.
+
+Bracket Syntax
+^^^^^^^^^^^^^^
+
+Using brackets, the code would be something like: ``object["key"]``. The only restraint in naming a key is that it has to be a valid JavaScript string.
+Since a key could potentially have a space in it, bracket syntax would be the only way to access the value in that property because of the quotes.
+
+Dot Notation
+^^^^^^^^^^^^
+
+With dot notation, the code would be something like: ``object.key``. Notice that the key is no longer surrounded by quotes. However, keys are still strings.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          name: "Pete",
+          weight: 919,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"]
+      };
+
+      console.log(tortoiseOne["name"]);
+      console.log(tortoiseOne.name);
+
+   **Console Output**
+
+   ::
+
+      Pete
+      Pete
+
+Modifying Properties
+--------------------
+
+A programmer can modify the value that a property holds by using either notation above.
+
+.. sourcecode:: js
+   :linenos:
+
+   let object = {
+       key1: value1,
+       key2: value2
+   };
+   
+   console.log(object.key1);
+
+   object.key1 = value3;
+
+   console.log(object["key1"]);
+
+Console Output
+
+::
+
+  value1
+  value3
+
+
+.. warning::
+ 
+   Recall that mutability means that a data structure can be modified without making a copy of that structure.
+   Objects are mutable data structures.
+   When you change the value of a property, the original object is modified and a copy is NOT made.
+
+.. admonition:: Example
+
+   In our zoo software, we may want to update Pete's weight as he has gained 10 lbs.
+   We will use both bracket syntax and dot notation for our software, but that is not a requirement!
+   Feel free to use whichever one suits your needs and is easiest for you and your colleagues to read.
+
+   .. sourcecode:: js
+      :linenos:
+
+      let tortoiseOne = {
+          species: "Galapagos Tortoise",
+          name: "Pete",
+          weight: 919,
+          age: 85,
+          diet: ["pumpkins", "lettuce", "cabbage"]
+      };
+
+      console.log(tortoiseOne.weight);
+
+      newWeight = tortoiseOne.weight + 10;
+
+      tortoiseOne["weight"] = newWeight;
+
+      console.log(tortoiseOne["weight"]);
+
+   **Console Output**
+
+   ::
+
+      919
+      929
+
+
+Check Your Understanding
+------------------------
+
+All of the questions below refer to an object called ``giraffe``. Here is the object literal for ``giraffe``.
```

Comment:
Word removal: These two sentences are redundant.  The same result can be achieved with "All of the questions below refer to an object called ``giraffe``:"